dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/webapi/json/JsonDocument.java

```diff
@@ -200,20 +200,33 @@ public final String toString()
         @Override
         public final JsonNode replaceWith( String json )
         {
-            int end = endIndex();
+            int endIndex = endIndex();
             StringBuilder newJson = new StringBuilder();
             if ( start > 0 )
             {
                 newJson.append( this.json, 0, start );
             }
             newJson.append( json );
-            if ( end < this.json.length )
+            if ( endIndex < this.json.length )
```

Comment:
Is this possibly always less than this.json.length because max index = length-1?